web/frontend/src/pages/ballot/components/Select.tsx

```diff
@@ -1,24 +1,24 @@
-import { FC } from 'react';
+import { FC, useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { Answers, SelectQuestion } from 'types/configuration';
 import { answersFrom } from 'types/getObjectType';
 import HintButton from 'components/buttons/HintButton';
-
 type SelectProps = {
   select: SelectQuestion;
   answers: Answers;
   setAnswers: (answers: Answers) => void;
+  language: string;
 };
 
-const Select: FC<SelectProps> = ({ select, answers, setAnswers }) => {
+const Select: FC<SelectProps> = ({ select, answers, setAnswers, language }) => {
   const { t } = useTranslation();
 
   const handleChecks = (e: React.ChangeEvent<HTMLInputElement>, choiceIndex: number) => {
     const newAnswers = answersFrom(answers);
     let selectAnswers = newAnswers.SelectAnswers.get(select.ID);
 
     if (select.MaxN === 1) {
-      selectAnswers = new Array<boolean>(select.Choices.length).fill(false);
+      selectAnswers = new Array<boolean>(select.ChoicesMap.get('en').length).fill(false);
```

Comment:
```suggestion
      selectAnswers = new Array<boolean>(select.ChoicesMap.get(language).length).fill(false);
```

or am I missing something ?

Reply:
still got the same remark